docs/wiki/research/scaling/eip-4844.md

```diff
@@ -0,0 +1,220 @@
+# EIP-4844
+
+[EIP-4844](https://eips.ethereum.org/EIPS/eip-4844) introduces Blobs, a new space to store data inside a transaction that will be much cheaper than `calldata` because they will have a new type of gas that is independent of normal gas and follows its own targeting rule.
```

Comment:
with the new type of gas, I would elaborate a little here that we are talking about different gas metering because nodes are handling this kind of data in different way, more effectively

Reply:
I've added more context around new gas handling on this commit [this commit](https://github.com/eth-protocol-fellows/protocol-studies/pull/51/commits/41f21a0c1f1ee6e389ba52752a943842119fd11c) please let know how it looks!